src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.csproj

```diff
@@ -6,7 +6,7 @@
     <MinVerTagPrefix>Instrumentation.ElasticsearchClient-</MinVerTagPrefix>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="System.Text.Json" Version="4.7.2" />
-    <PackageReference Include="OpenTelemetry" Version="0.7.0-beta.1" />
+    <PackageReference Include="OpenTelemetry" Version="1.0.1" />
```

Comment:
need to goto a version which has https://github.com/open-telemetry/opentelemetry-dotnet/pull/1836 changes , as thats the only which eliminates ActivitySourceAdapter.

Reply:
we'll have it on myget soon, will validate and release to nuget after enough validations.

Reply:
@cijothomas I'm not sure what that means. I see that you guys are working on removing `ActivitySourceAdapter`, but I'm not using it and I'm using a similar method as what is being done in that PR. I did notice the `AddLegacyOperationName` ability, but I don't see how you go about enriching the activities like I'm doing in this instrumentation when using that method.

Reply:
Not to mention, that I see that you are STILL using a BUNCH of internal methods and utilities in your own instrumentations even after this https://github.com/open-telemetry/opentelemetry-dotnet/pull/1836 is merged. So I really don't see how that PR is going to help me much. If I just comment out the `InternalsVisibleTo` for your `GrpcNetClient` in this latest code there are still a bunch of things I can't do:

![image](https://user-images.githubusercontent.com/282584/109246847-90a6e580-77a8-11eb-9956-b2a393875a4b.png)

I really don't understand why you wouldn't get rid of those `InternalsVisibleTo` and stop cheating so you can see what it's like for contributors to try and create their own instrumentations.

Reply:
> @cijothomas I'm not sure what that means. I see that you guys are working on removing `ActivitySourceAdapter`, but I'm not using it and I'm using a similar method as what is being done in that PR. I did notice the `AddLegacyOperationName` ability, but I don't see how you go about enriching the activities like I'm doing in this instrumentation when using that method.

With using `AddLegacyOperationName`, the activity created in ES library is not going through the samplers registered in the TracerProvider.

Reply:
The internalsvisible is now only used for helpers. It can be copied by any project. (even in the main repo, they can be copied to individual instrumentations and remove internalvisible attribute. Feel free to submit a PR to restructure it. if you think its important.)

`ActivitySourceAdapter` is/was the only one which was specific to the OTel SDK, and folks outside that repo could not have just copied it over - it was done to unblock progress at the time, and now its being removed.

Reply:
@cijothomas done https://github.com/open-telemetry/opentelemetry-dotnet/pull/1856